CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+ - Fixed-Point instructions were added (`VSADDU`, `VSADD`, `VAADD`, `VAADDU`, `VSSUBU`, `VSSUB`, `VASUB`, `VASUBU`)
```

Comment:
Please use the present tense (`Add fixed-point instructions`). I also think this would be better suited for the `Added` section, instead of `Changed` section.